app-eselect/eselect-wxwidgets/files/0001-wxwidgets.eselect-multilib-support.patch

```diff
@@ -0,0 +1,277 @@
+From 69efadbccf10da6e65c7d459c5cfea38375824b6 Mon Sep 17 00:00:00 2001
+From: nE0sIghT <ykonotopov@gmail.com>
+Date: Sun, 29 Mar 2015 21:18:56 +0300
+Subject: [PATCH] wxwidgets.eselect: multilib support
+
+---
+ modules/wxwidgets.eselect | 181 +++++++++++++++++++++++++++++-----------------
+ 1 file changed, 116 insertions(+), 65 deletions(-)
+
+diff --git a/modules/wxwidgets.eselect b/modules/wxwidgets.eselect
+index 55d49d2..04ad16b 100644
+--- a/modules/wxwidgets.eselect
++++ b/modules/wxwidgets.eselect
+@@ -6,10 +6,11 @@ inherit config
+ 
+ DESCRIPTION="Manage the system default wxWidgets profile"
+ MAINTAINER="wxwidgets@gentoo.org"
+-VERSION="20140423"
++VERSION="20150329"
+ 
+ WXCONFFILE="${EROOT}"/var/lib/wxwidgets/current
+-WXCONFDIR="${EROOT}"/usr/lib/wx/config
++WXCONFDIR_64="${EROOT}"/usr/lib/wx/config
++WXCONFDIR_32="${EROOT}"/usr/lib32/wx/config
```

Comment:
No, no, no. There's some eselect library for multilib support. If you're doing that, you need to support all ABIs, not just x86+amd64.